src/main/java/net/neoforged/neoform/runtime/cli/RunNeoFormCommand.java

```diff
@@ -504,4 +483,47 @@ private static ReplaceNodeOutput.NodeFactory sourceTransform(NeoFormEngine engin
             return builder.output("output", NodeOutputType.ZIP, "Sources with additional transforms (ATs, Parchment, Interface Injections) applied");
         };
     }
+
+    private static ApplyDevTransformsAction getOrAddDevTransformsAction(NeoFormEngine engine) {
+        return (ApplyDevTransformsAction) getOrAddDevTransformsNode(engine).action();
+    }
+
+    private static ExecutionNode getOrAddDevTransformsNode(NeoFormEngine engine) {
+        var graph = engine.getGraph();
+        var transformNode = graph.getNode("applyDevTransforms");
+        if (transformNode != null) {
+            if (transformNode.action() instanceof ApplyDevTransformsAction) {
+                return transformNode;
+            } else {
+                throw new IllegalStateException("Node applyDevTransforms has a different action type than expected. Expected: "
+                        + ApplyDevTransformsAction.class + " but got " + transformNode.action().getClass());
+            }
+        }
+
+        NodeOutput untransformedOutput;
+        if (!engine.getProcessGeneration().sourcesUseIntermediaryNames()) {
+            untransformedOutput = engine.getGraph().getResult(ResultIds.GAME_JAR_NO_RECOMP);
+        } else {
+            // We have to transform in srg
+            var remapSrgClasses = engine.getGraph().getNode("remapSrgClassesToOfficial");
+            if (remapSrgClasses == null || !(remapSrgClasses.getRequiredInput("input") instanceof NodeInput.NodeInputForOutput nifo)) {
+                throw new IllegalStateException("Could not find SRG input to apply dev transform to.");
+            }
+            untransformedOutput = nifo.getOutput();
+        }
+
+        new ReplaceNodeOutput(
+                untransformedOutput.getNode().id(),
+                untransformedOutput.id(),
+                "applyDevTransforms",
+                (builder, previousOutput) -> {
+                    builder.input("input", previousOutput.asInput());
+                    var transformedOutput = builder.output("output", NodeOutputType.JAR, "The jar file with the desired dev transforms applied.");
+                    builder.action(new ApplyDevTransformsAction());
+
+                    return transformedOutput;
+                }).apply(engine, graph);
```

Comment:
While that works, please use engine.applyTransform(...) instead so we can keep doing find usage on it.

Reply:
Oh well that's already in main :-|